app/views/lookup_keys/_fields.html.erb

```diff
@@ -4,9 +4,9 @@
   <%= remove_child_link(_("Remove %s?") % (f.object.new_record? ? _("Variable") : f.object), f, { :class => 'btn btn-danger hide' }) if controller_name ==  "puppetclasses" && !is_param %>
   <fieldset>
     <h2><%= _("Parameter details") %></h2>
-    <%= text_f f, :key, :disabled => is_param, :size => "col-md-8" %>
+    <%= text_f f, :key, :disabled => is_param, :size => "col-md-8", :label => _('Key') %>
```

Comment:
No need to extract the strings again here, or even translate them, the form helpers do it automatically. They're likely to get out of sync.